framework/src/interfaces/TaggingInterface.C

```diff
@@ -359,21 +381,73 @@ TaggingInterface::prepareMatrixTagLower(Assembly & assembly,
 void
 TaggingInterface::accumulateTaggedLocalResidual()
 {
-  for (auto & re : _re_blocks)
-    *re += _local_re;
-  for (auto & absre : _absre_blocks)
-    for (const auto i : index_range(_local_re))
-      (*absre)(i) += std::abs(_local_re(i));
+  if (_bx_norm)
+  {
+    DenseVector<Number> scaled_local_re = _local_re;
+    if (_eigen_problem->negativeSignEigenKernel())
+      scaled_local_re.scale(1 / (*_bx_norm));
+    else
+      scaled_local_re.scale(-1 / (*_bx_norm));
+
+    for (MooseIndex(_re_blocks) i = 0; i < _re_blocks.size(); i++)
+      if (_scale_by_bx_norm[i])
+        *_re_blocks[i] += scaled_local_re;
+      else
+        *_re_blocks[i] += _local_re;
+
+    for (MooseIndex(_absre_blocks) i = 0; i < _absre_blocks.size(); i++)
```

Comment:
Use `index_range` instead of `MooseIndex`

Reply:
This line no longer exists in the new implementation.